lib/drone.rb

```diff
@@ -11,23 +23,22 @@ def translate_status_to_class(status)
 end
 
 def get_latest_build_for_each_branch(json)
-  json.group_by{|b| b['branch']}.map{|branch| branch[1].max_by{|build| build['started_at']}}
+  json.group_by { |b| b['branch'] }.map { |branch| branch[1].max_by { |build| build['started_at'] } }
 end
 
 def get_drone_builds_for_repo(repo_name)
-  uri = URI.parse('https://drone.digital.homeoffice.gov.uk/api/repos/UKHomeOffice/' + repo_name + '/builds')
-  drone_results = Net::HTTP.get(uri)
-  parsed_results = JSON.parse(drone_results)
+  response = HTTParty.get('https://drone.digital.homeoffice.gov.uk/api/repos/' + repo_name + '/builds')
```

Comment:
This is just a note more than a suggestion. But this assumes github+drone. Some of the delivery teams use gitlab+drone. In the latter the url is not `https://drone.digital...` but `https://drone-gitlab.digital...`.